src/remote-component.config.js

```diff
@@ -3,6 +3,10 @@
  */
  module.exports = {
   resolve: {
-    react: require("react")
+    react: require("react"),
+    '@chakra-ui/react': require('@chakra-ui/react'),
+    '@emotion/react': require('@emotion/react'),
+    '@emotion/styled': require('@emotion/styled'),
+    'framer-motion': require('framer-motion')
```

Comment:
Just making a note here: basically eventually we would want people to choose which packages are bundled into their lenses as opposed to forcing them to bundle these. Need to figure out how that could be done.

Reply:
Agreed! Probably with the exception of React itself, for now.

This'll mean we ultimately want to remove the Chakra provision for the doc component, too.

Reply:
Actually, now that I think of it, this doesn't actually force these packages on lens creators. This just exposes packages we're already using in the Tiles app so that other people can use them.

More of an issue might be that people want to use systems which aren't compatible with Chakra, and anything else we're passing to the component. This echoes what I said above about removing ChakraProvider so that all lens components operate in a neutral environment.

Even still, I suspect lens creators may still want to go as raw as possible and go with whatever TIles has in the app, because this reduces the bundle size of the remote lens.